extensions/amp-fit-text/1.0/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@ampproject/amp-fit-text",
```

Comment:
Is it allowed to have more complicated names here? E.g. `@ampproject/components/amp-fit-text`?

Reply:
No, you're only allowed `\w+` or `@\w+/\w+`. If we wanted all components to go into `@ampproject/components/*`, we would have to publish a `@ampproject/components` package. That's possible, but we'd have complications with consuming different versions of our extensions.

Reply:
Yes. We definitely agreed we won't do that. So let's close this thread then.